apps/users/models.py

```diff
@@ -484,6 +484,24 @@ def remove_locale(self, locale):
     def get_fallback(cls):
         return cls._meta.get_field('lang')
 
+    @amo.cached_property
+    def mobile_addons(self):
+        from bandwagon.models import CollectionAddon  # Import loop.
+        return CollectionAddon.objects.filter(
+            collection__author=self, collection__type=amo.COLLECTION_MOBILE)
```

Comment:
Are there no reverse-relations that you could use instead of importing? E.g `self.collections.filter(type=amo.COLLECTION_MOBILE)`

Same for the both methods below.